### PR TITLE
closes #52 add new database items to seed file

### LINKED ISSUE #52: Add New Database Items to Seed File

### PR DESCRIPTION
**Pull request by ToniRib**
We now have a giant seeds.rb file!!

I tried to follow the requirements given for total # of business (job listers), categories, etc. that are in the pivot project requirements.

This pull request also sets up the main model relationships since they were necessary to actually create this file. There's some cool stuff in here like making sure that for any jobs that are "in progress" or "completed" that there is one accepted bid and all others are rejected. Additionally, for any "completed" job, both required comments have been seeded.

Take a look and let me know what you think!

### CODE REVIEW

**Comment by ToniRib** [comment]
## Don't merge this yet, I'm working on an issue!

**Comment by ToniRib** [comment]
Ignoring the long lines because I think they read more easily this way. Also, the bug fix I had to put in was for jobs that received no bids. In this case, I set their status to "cancelled" so we don't try to apply comments to them.

File: app/models/bid.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 class Bid < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :job
+
+  enum status: %w(pending accepted rejected)
```
Comment (danjwinter) [inline app/models/bid.rb:5]: As we discussed earlier about the possibility of a cancelled status for bids, it makes more sense that if a contractor wants to remove their bid, it should be removed and there is no current business need to hold on to that information in the form of a cancelled status.

File: app/models/user.rb [hunk 2]
```diff
@@ -21,7 +25,8 @@ class User < ActiveRecord::Base
   validates_attachment_content_type :file_upload,
                                     content_type: %r{\Aimage\/.*\Z}
 
-  scope :lister, -> { where(role: 1) }
+  scope :listers, -> { where(role: 1) }
+  scope :contractors, -> { where(role: 0) }
```
Comment (danjwinter) [inline app/models/user.rb:29]: Great call on adding all the scopes!

File: db/seeds.rb [hunk 3]
```diff
@@ -20,28 +26,190 @@ def create_categories
...
+        last_name: last_name,
+        username: "#{first_name.downcase}_user",
+        password: "password",
+        role: 0,
+        email_address: "#{first_name.downcase}@turing.io",
+        street_address: Faker::Address.street_address,
+        city: Faker::Address.city,
+        state: Faker::Address.state,
+        zipcode: Faker::Address.zip,
+        bio: Faker::Lorem.sentence(3)
```
Comment (danjwinter) [inline db/seeds.rb:57]: Thanks for adding the Faker aspect, it'll make for MUCH better seeds.

**Comment by danjwinter** [comment]
Seed file and relationships lookin' good.

**Pull request merged**